packages/next-codemod/bin/cli.ts

```diff
@@ -108,6 +108,12 @@ export function runTransform({ files, flags, transformer }) {
       )
     }
   }
+
+  if (!dry && transformer === 'next-request-geo-ip') {
+    console.log('Note: This Codemod expects your app to be deployed on Vercel.')
```

Comment:
`expects your app to be deployed on Vercel.` this might be too strict, users could run this codemod and have no files changed. We don't need to mention it. If there's matched, they were already using this feature